docs-client/webpack.config.ts

```diff
@@ -124,19 +126,86 @@ plugins.push(new FaviconsWebpackPlugin({
   mode: 'light',
   devMode: 'light',
 }));
+
 // Do not add LicenseWebpackPlugin on Windows, because otherwise it will fail with a known issue.
 if (!isWindows) {
   plugins.push(new LicenseWebpackPlugin({
     stats: {
       warnings: true,
       errors: true,
     },
+    excludedPackageTest: (packageName: string) => {
+      // @monaco-editor/react and monaco-editor are the same package and causes license duplication.
+      return packageName.includes("monaco-editor");
+    },
```

Comment:
@trustin @ikhoon @jrhee17 

As a workaround we can add a separate file `web-licenses-extra.txt` and copy paste licenses from the `monaco-editor` packages to unblock ourselves. Wdy think?

Reply:
Did you try running with `perChunkOutput:`false` by any chance?   I don't think we want to generate license files per chunk anyways and it seemed to run without errors in my local env.

ref: https://github.com/xz64/license-webpack-plugin/blob/master/DOCUMENTATION.md

Reply:
That + @ikhoon 's changes regarding the number of chunks actually fixed the error and now we have a lot more licenses inside the license file. Probably it was what it should have been in the first place.